test/fulfillment_v2_test.rb

```diff
@@ -0,0 +1,91 @@
+require 'test_helper'
+
+class FulfillmentV2Test < Test::Unit::TestCase
+  def setup
+    super
+    fake "fulfillment_orders/519788021", method: :get,
+         body: load_fixture('fulfillment_order')
+  end
+
+  context "FulfillmentOrderFulfillment" do
+    context "#create" do
+      should "be able to create a fulfillment order fulfillment" do
+        create_fulfillment_attributes = {
+          message: "The message for this FO fulfillment",
+          notify_customer: true,
+          tracking_info: {
+            number: "XSDFHYR23475",
+            url: "https://tracking.example.com/XSDFHYR23475",
+            company: "TFTC - the fulfillment/tracking company"
+          },
+          line_items_by_fulfillment_order: [
+            {
+              fulfillment_order_id: 3,
+              fulfillment_order_line_items: [{ id: 2, quantity: 1 }]
+            }
+          ]
+        }
+        request_body = { fulfillment: create_fulfillment_attributes }
+        response_body = { fulfillment: create_fulfillment_attributes.merge(id: 346743624) }
+        fake "fulfillments", :method => :post,
+          :request_body => ActiveSupport::JSON.encode(request_body),
+          :body => ActiveSupport::JSON.encode(response_body)
+
+        fulfillment = ShopifyAPI::FulfillmentV2.new(create_fulfillment_attributes)
+        assert fulfillment.save
+        assert_equal 346743624, fulfillment.id
+      end
+    end
+
+    context "#update_tracking" do
+      should "be able to update tracking info for a fulfillment" do
+        tracking_info = {
+          number: 'JSDHFHAG',
+          url: 'https://example.com/fulfillment_tracking/JSDHFHAG',
+          company: 'ACME co',
+        }
+        fake_f = ActiveSupport::JSON.decode(load_fixture('fulfillment'))['fulfillment']
```

Comment:
nit: I think a more verbose variable would increase the readability of these tests. Maybe `fake_fulfillment` or `resulting_fulfillment`?